tests/unit/test_utils.py

```diff
@@ -0,0 +1,20 @@
+import pytest
+
+from amazon_transcribe.utils import ensure_boolean
+
+
+class TestEnsureBoolean:
```

Comment:
This doesn't necessarily need a change, but as a general structural note, you typically only need to create a test class if you want to bundle tests together around specific functionality. If we don't expect to add a lot more tests around `ensure_boolean`, this would just be a stand alone test function.

Reply:
Gotcha, I was looking at how it was done in botocore